src/main/java/duke/Parser.java

```diff
@@ -16,6 +16,9 @@ public static Command parse(String fullCommand) throws EmptyDescriptionException
             case ("done"):
                 int index = Integer.parseInt(fullCommand.split(" ")[1]) - 1;
```

Comment:
Should `catch` or `throw` `NumberFormatException` if the input is non-numerical.